session/persistence.go

```diff
@@ -75,5 +78,17 @@ func TestPersister(p interface {
 			_, err := p.GetSession(context.Background(), expected.ID)
 			require.Error(t, err)
 		})
+
+		t.Run("case=delete session for", func(t *testing.T) {
+			var expected Session
+			require.NoError(t, faker.FakeData(&expected))
+			require.NoError(t, p.CreateIdentity(context.Background(), expected.Identity))
+			require.NoError(t, p.CreateSession(context.Background(), &expected))
+			require.NoError(t, p.CreateSession(context.Background(), &expected))
```

Comment:
Here too, it would be better to create two individual sessions and create them :) That ensures that really ALL sessions are revoked, not just one